src/assessment/builder.py

```diff
@@ -504,7 +509,10 @@ def build_emails(domains, labels):
                     logging.error(
                         "{} Domain Miss Match Errors".format(len(format_error))
```

Comment:
Typo:
```suggestion
                        "{} Domain Mismatch Errors".format(len(format_error))
```

Reply:
Sharp eyes, @dav3r!

Reply:
Accepting fix. Thanks for catching that!!